rotkehlchen/tasks/calendar.py

```diff
@@ -410,6 +422,57 @@ def maybe_create_airdrop_claim_reminder(self) -> None:
             error_msg='Failed to add the airdrop claim reminders',
         )
 
+    def maybe_create_l2_bridging_reminder(self) -> None:
+        """Creates calendar reminders for L2 bridge claims (7 days after deposit).
+        Tracks deposits on Base, Optimism, and Arbitrum networks.
+        """
+        bridge_events: list[EvmEvent] = []
+        locations_to_counterparties = (
+            (Location.BASE, CPT_BASE),
+            (Location.OPTIMISM, CPT_OPTIMISM),
+            (Location.ARBITRUM_ONE, CPT_ARBITRUM_ONE),
+        )
+        db_history_events = DBHistoryEvents(database=self.database)
+        with self.database.conn.read_ctx() as cursor:
+            for location, counterparty in locations_to_counterparties:
+                bridge_events.extend(db_history_events.get_history_events(
+                    cursor=cursor,
+                    has_premium=True,
+                    filter_query=EvmEventFilterQuery.make(
+                        and_op=True,
+                        location=location,
+                        counterparties=[counterparty],
+                        event_types=[HistoryEventType.DEPOSIT],
+                        event_subtypes=[HistoryEventSubType.BRIDGE],
+                    ),
+                ))
+
+        now = ts_now_in_ms()
+        bridge_calendar_entries: list[int] = []
+        for bridge_event in bridge_events:
+            if now - bridge_event.timestamp > WEEK_IN_SECONDS * 1000:
```

Comment:
Why not jsut use ts_now()?

```suggestion
            if ts_now() - bridge_event.timestamp > WEEK_IN_SECONDS:
```

Reply:
`bridge_event.timestamp` is in milliseconds. also why `now = ts_now_in_ms()` is defined outside of the for-loop.